### PR TITLE
LibJS: Add BigInt64Array and BigUint64Array

### PR DESCRIPTION
**Pull request by Lubrsi**
This fixes ~297 test262 test cases :^)
More details in the individual commits.

### CODE REVIEW

File: Userland/Libraries/LibJS/Runtime/Value.cpp [hunk 1]
```diff
@@ -548,6 +548,25 @@ BigInt* Value::to_bigint(GlobalObject& global_object) const
...
 }
 
+// 7.1.15 ToBigInt64 ( argument ), https://tc39.es/ecma262/multipage/abstract-operations.html#sec-tobigint64
+i64 Value::to_bigint_int64(GlobalObject& global_object) const
+{
+    auto* bigint = to_bigint(global_object);
+    if (global_object.vm().exception())
+        return INVALID;
+    // FIXME: This is probably wrong.
+    return static_cast<i64>(bigint->big_integer().to_u64());
```
Comment (IdanHo) [inline Userland/Libraries/LibJS/Runtime/Value.cpp:558]: this is not wrong. `to_u64` of SignedBigInteger returns a 2nd complement representation of the number for the negative case, making this cast always correct.

Comment (Lubrsi) [reply Userland/Libraries/LibJS/Runtime/Value.cpp]: Cool, fixed :+1:

**Review by linusg — COMMENTED** [review]
Awesome! I think we can remove all the `// Update when more typed arrays get added` comments now, seems complete?

**Comment by Lubrsi** [comment]
> Awesome! I think we can remove all the `// Update when more typed arrays get added` comments now, seems complete?

Sure, done :+1:

**Review by linusg — APPROVED** [review]

**Pull request merged**